wled00/data/simple.css

```diff
@@ -19,19 +19,13 @@
 	--c-r: #e42;
 	--c-g: #4e2;
 	--c-l: #48a;
-	--t-b: 0.5;
-	--c-o: rgba(34, 34, 34, 0.9);
-	--c-tb : rgba(34, 34, 34, var(--t-b));
-	--c-tba: rgba(102, 102, 102, var(--t-b));
-	--c-tbh: rgba(51, 51, 51, var(--t-b));
-	/*following are internal*/
+	--c-o: rgba(34, 34, 34, .9);
+	--c-tb : rgba(34, 34, 34, .5);
+	--c-tba: rgba(102, 102, 102, .5);
+	--c-tbh: rgba(51, 51, 51, .5);
 	--th: 70px;
 	--tp: 70px;
 	--bh: 63px;
-	--tbp: 14px 8px 10px;
```

Comment:
Why do you want to remove these?

Reply:
Optimisation - it's a process overhead (small) and filesize overhead (small too) for no point. It's also a developer readability overhead. These variables are only uses once. 

If they were reused multiple times - I'd have left them in.

Reply:
What makes you say that? Have you combed the sources?
They get modified during runtime by JS.

Reply:
Agghh yuk... I see that now. For the changes between light & dark mode... not nice that this is done in JS (it can/should be done solely in CSS).

Thank you 🙇🏼 for pointing that out (it's good to have a Maintainer that's aware of all the parts of the project).

I'm working on a new "look" 💅🏼 for the Web UI and will aim to remove/reduce these.